internal/operator/builder.go

```diff
@@ -198,6 +206,7 @@ func (b *Builder) Build(ctx context.Context) (manager.Manager, error) {
 			HealthProbeBindAddress: b.probeAddress,
 			LeaderElection:         b.leaderElection,
 			LeaderElectionID:       b.leaderElectionID,
+			Client:                 client.Options{DryRun: pointer.MakePtr(b.dryRun)},
```

Comment:
q: given we need to run reconcilers manually, are we actually going to leverage the manager for dry-running?